src/odemis/gui/win/acquisition.py

```diff
@@ -612,6 +613,409 @@ def on_acquisition_done(self, future):
         self.btn_secom_acquire.Enable()
 
 
+class OverviewAcquisitionDialog(xrcfr_overview_acq):
+    """
+    Class used to control the cryo-secom overview dialog
+    """
+    def __init__(self, parent, orig_tab_data):
+        xrcfr_overview_acq.__init__(self, parent)
+
+        self.conf = get_acqui_conf()
+
+        # True when acquisition occurs
+        self.acquiring = False
+        self.data = None
+
+        # a ProgressiveFuture if the acquisition is going on
+        self.acq_future = None
+        self._acq_future_connector = None
+
+        self._main_data_model = orig_tab_data.main
+
+        # duplicate the interface, but with only one view
+        self._tab_data_model = self.duplicate_tab_data_model(orig_tab_data)
+
+        self.filename = model.StringVA(create_filename(self.conf.last_path, self.conf.fn_ptn,
+                                                       self.conf.last_extension, self.conf.fn_count))
+        self.filename.subscribe(self._onFilename, init=True)
```

Comment:
There shouldn't be the need to select a filename. It should be automatically decided. It always goes into the "project" folder (which is selected in the chamber tab), and then I'd suggest to use a fixed filename pattern like "yyyymmdd-hhmmss-overview.ome.tiff". It's always a TIFF. As the chamber tab is not yet ready. I'd suggest to use "last_path" as folder.